artman/conductors/cloudtasks_conductor.py

```diff
@@ -120,14 +120,14 @@ def _delete_task(task_client, task):
 
 def _setup_logger(log_path):
     """Setup logger with one-time logging FileHandler."""
-    logger = logging.getLogger('artman')
     log_file_handler = logging.FileHandler(log_path)
     logger.addHandler(log_file_handler)
     return log_file_handler
 
 
 def _write_to_cloud_logging(log_id, log_file_path):
     """Write log file content to cloud logging"""
+    global CLOUD_LOGGING_CLIENT
```

Comment:
Why global? Why is conductor not an object that has a logging client as an instance variable?